db/schema.rb

```diff
@@ -0,0 +1,44 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_02_19_202916) do
+
+  create_table "comments", force: :cascade do |t|
+    t.string "comment"
```

Comment:
[OPTIONAL]
This name is fine, but it could possibly be confusing in app development if this were to be developed more.
It'd be better to not name it after the name of the model. Something like `body` or `content` may be more suitable.

Reply:
We will generate a migration to fix the same. Thank you for pointing this out, it's very appreciated.